Packs/DynamicSectionReports/README.md

```diff
@@ -0,0 +1,9 @@
+Dynamic sections allow you to populate content into indicator and incident layouts using an automation script.
+The `DisplayIndicatorReputationContent` automation contained in this pack makes it easy to display all reputation information collected for a particular indicator directly on its layout.
+
+<img src="https://github.com/demisto/content/raw/0ab37a70cd964e903a8fae02501bff3577c1fc67/Packs/DynamicSectionReports/doc_files/indicator_report.png" width="500px" />
```

Comment:
can you share a print screen? I suggest using relative paths, https://xsoar.pan.dev/docs/documentation/readme_file#relative-image-urls
but your call.

Reply:
will take a look and update, thanks!

Reply:
Ahh I did try to use relative links to start but I received error in the linter

```
  Packs/DynamicSectionReports/README.md: [RM108] - Error in readme image:
Detected the following image relative path: (doc_files/indicator_report.png).
Relative paths are not supported in pack README files. See https://xsoar.pan.dev/docs/integrations/integration-docs#images for further info on how to add images to pack README files.
```

Reply:
sorry, it is supported only for non pack readme files.